pom.xml

```diff
@@ -5,9 +5,37 @@
     <artifactId>swim</artifactId>
     <version>1.0-SNAPSHOT</version>
     <packaging>jar</packaging>
+    
+    
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>1.4.2.RELEASE</version>
+    </parent>
```

Comment:
Notice how we are doing a spring boot application.  The way it works that all spring boot projects are a sub project of spring boot.  The parent then has all the necessary info for dependencies that spring is familiar with and is designed to work with seamlessly (configurations and conventions)